iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -250,4 +259,43 @@ private static PartitionSpec spec(Schema schema, Properties properties,
       return PartitionSpec.unpartitioned();
     }
   }
+
+  @Override
+  public void commitInsertTable(org.apache.hadoop.hive.metastore.api.Table table, boolean overwrite)
+      throws MetaException {
+    // construct the job context
+    JobConf jobConf = new JobConf(conf);
+    String tableName = TableIdentifier.of(table.getDbName(), table.getTableName()).toString();
```

Comment:
How this will get the `Catalog` information?

Reply:
Not sure yet, I'll look into it. The catalog name might be stored in the table params, or we might need to load the Iceberg table to find out if we can. @lcspinter might have some more insights